core/rawdb/database_ext.go

```diff
@@ -0,0 +1,71 @@
+// (c) 2025, Ava Labs, Inc.
```

Comment:
seems should match the 2 line format for the copyright header
(also in other places)

Reply:
Good point!
We definitely should setup goheader in our CI (later, probably after the subnet-evm/coreth merge!)